card.css

```diff
@@ -0,0 +1,130 @@
+.card-container {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.7rem;
+}
+.vertical-card {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  position: relative;
+  max-width: 19rem;
+  text-align: center;
+  margin: auto;
+  margin-top: 0.6rem;
+  cursor: pointer;
+}
+.vertical-card img {
+  max-width: 100%;
+  height: 19rem;
+}
+.add-to-cart-btn {
+  border: none;
+  outline: 0;
+  padding: 0.6rem;
+  color: var(--white);
+  background-color: var(--secondary-color);
+  text-align: center;
+  cursor: pointer;
```

Comment:
you can give the button cursor pointer in utility class, so you wont have to right repeating code.

Reply:
Okay